$MirOS: ports/emulators/qemu/snapshot/patches/patch-cutils_c,v 1.1 2008/05/07 15:53:09 tg Exp $
$OpenBSD: patch-cutils_c,v 1.2 2008/04/28 22:52:38 todd Exp $
--- cutils.c.orig	Wed Sep 23 19:01:08 2009
+++ cutils.c	Sat Oct 17 16:35:43 2009
@@ -24,7 +24,7 @@
 #include "qemu-common.h"
 #include "host-utils.h"
 
-void pstrcpy(char *buf, int buf_size, const char *str)
+void pstrcpy(char *buf, size_t buf_size, const char *str)
 {
     int c;
     char *q = buf;
@@ -42,7 +42,7 @@ void pstrcpy(char *buf, int buf_size, co
 }
 
 /* strcat and truncate. */
-char *pstrcat(char *buf, int buf_size, const char *s)
+char *pstrcat(char *buf, size_t buf_size, const char *s)
 {
     int len;
     len = strlen(buf);
